app/src/app/components/contents/PacketGroup/hooks/useGetPacketPages.ts

```diff
@@ -3,14 +3,16 @@ import appConfig from "../../../../../config/appConfig";
 import { fetcher } from "../../../../../lib/fetch";
 import { PageablePackets } from "../../../../../types";
 
-export const useGetPacketGroups = (packetName: string | undefined, pageNumber: number, pageSize: number) => {
+export const useGetPacketPages = (packetGroupName: string | undefined, pageNumber: number, pageSize: number) => {
```

Comment:
Note to self, file name is not in sync with function name. In the back end this queries on packets so function rename is questionable.